nailgun/entities.py

```diff
@@ -4302,6 +4303,64 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         return super(OverrideValue, self).read(entity, attrs, ignore, params)
 
 
+class Parameter(
+        Entity,
+        EntityCreateMixin,
+        EntityReadMixin):
+    """A representation of a Parameter entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'name': entity_fields.StringField(
+                required=True,
+                str_type='alpha',
+                length=(6, 12),
+            ),
+            'priority': entity_fields.IntegerField(),
+            'value': entity_fields.StringField(required=True),
+        }
+        self._path_fields = {
```

Comment:
Have an idea btw - it's not the first time we have some fields which are used in path and are forced to override `read()` to ignore such fields since they're never returned. For such cases we could define some `self._path_fields` or `self._ignored_fileds` which we could ignore in `EntityReadMixin.read()` with no need to reimplement specific entity's `read()` every single time.